src/components/base/outline-image/outline-image.css

```diff
@@ -1,33 +1,52 @@
 :host {
-  @apply block;
+  display: block;
 
-  &([parallax]) figure {
-    @apply h-half-screen-h;
-    clip-path: inset(0 0 0 0);
-  }
-
-  &([parallax='false']) figure {
-    @apply h-auto;
-    clip-path: inset(0 0 0 0);
+  figure {
+    margin: 0;
+    display: block;
+    height: auto;
   }
 }
 
-@screen lg {
-  :host {
-    &([parallax='false']) figure {
-      @apply h-auto;
-    }
-  }
+figure {
+  width: 100%;
+  height: 100%;
+  display: flex;
 }
 
-figure {
-  @apply m-0 block;
+picture {
+  aspect-ratio: default;
+  overflow: hidden;
+  display: flex;
 }
 
-::slotted([slot='caption']) {
-  @apply mt-2 font-body text-sm;
+img {
+  object-fit: cover;
+  height: auto;
+  min-width: 100%;
 }
 
-::slotted(img) {
-  @apply w-full;
+:host([image-ratio='1/1']) picture {
```

Comment:
Should `outline-image` receive attributes for image-ratio, and set the `aspect-ratio` accordingly?

Reply:
It does, but there's a finite set in CSS currently. It could be put in the TS file and generated dynamically based on config. I was considering that, but that's a tomorrow feature.